cmd/kaniko-docker/main.go

```diff
@@ -353,6 +353,11 @@ func main() {
 			Usage:  "Path to ignore during the build.",
 			EnvVar: "PLUGIN_IGNORE_PATH",
```

Comment:
Can we support comma-separated paths with this existing env variable instead of creating a new one?

Reply:
that required changes in all the supported versions of Kaniko and done - please check